code/Equipment/Gadgets/Projectiles/Shrapnel.cs

```diff
@@ -12,6 +14,7 @@ public sealed class Shrapnel : Component
 	[Property] public float ShrapnelUpVelocity { get; set; } = 500f;
 
 	[Property] public float ShrapnelSpreadVelocity { get; set; } = 150f;
+	[Property] public float ShrapnelSpawnRandomness { get; set; } = 0;
```

Comment:
Give a reasonable spawn randomness, and definitely include `0f` instead of `0`

Reply:
He put 0 so existing shrapnel doesn't get added randomness, this makes sense, not all shrapnel wants the spawn randomness I think.